socorro/external/rabbitmq/test_rabbitmq.py

```diff
@@ -1,6 +1,6 @@
 import pika
-import logging
-logging.basicConfig()
+##import logging
+##logging.basicConfig()
```

Comment:
yeah, I saw that in the orignals and realized that it just doubled each logged line.  I commented them out, confirming that fixed the logging, but didn't remember to go back to complete the job.